lerna.json

```diff
@@ -15,7 +15,6 @@
     "security/*",
     "longhaultests",
     "e2etests",
-    "edge-e2e/wrapper/nodejs-server-server",
```

Comment:
>"edge-e2e/wrapper/nodejs-server-server", [](start = 4, length = 40)

Question: do we also need to remove this from any version bump scripts?

Reply:
No.